maat-court-data-api/src/main/java/gov/uk/courtdata/model/assessment/CreatePassportAssessment.java

```diff
@@ -0,0 +1,22 @@
+package gov.uk.courtdata.model.assessment;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+import lombok.NoArgsConstructor;
+import lombok.experimental.SuperBuilder;
+
+import java.time.LocalDateTime;
+
+@Data
+@SuperBuilder
+@AllArgsConstructor
+@NoArgsConstructor
+@EqualsAndHashCode(callSuper = true)
+public class CreatePassportAssessment extends PassportAssessment {
+    private Integer financialAssessmentId;
```

Comment:
I can't see why this field is required? If associated financial assessments need updating then we can do so using the RepID.

Reply:
This is the reason :  
update TOGDATA.HARDSHIP_REVIEWS hr
         set
            HR.REPLACED = 'Y'
         where
            HR.FIAS_ID  = p_application_object.current_assessment_object.fin_assessment_object.**fin_assessment_id**;

Reply:
Well spotted. I wonder if there's a way to retrieve the latest financial assessment without needing it? Best to leave it for now though I think.